src/components/views/EResource.js

```diff
@@ -28,6 +28,8 @@ const propTypes = {
   handlers: PropTypes.shape({
     onClose: PropTypes.func.isRequired,
     onEdit: PropTypes.func.isRequired,
+    onNeedMoreEntitlements: PropTypes.func.isRequired,
+    onNeedMoreEntitlementOptions: PropTypes.func.isRequired,
```

Comment:
We can take the `PropTypes` off here for these 2 handlers as we arent using those within this file? You can cleanup the `onNeedMorePackageContents` propType as well